src/store/embalm/reducer.ts

```diff
@@ -38,6 +36,7 @@ export interface EmbalmState {
   requiredArchaeologists: string;
   resurrection: number;
   resurrectionRadioValue: string;
+  customResurrectionDate: Date | null;
```

Comment:
this embalm state is getting real juicy -- may want to consider at some point turning this into child-interfaces (with `EmbalmState` being the parent), if there are any patterns where groupings make sense (may not be)

Reply:
I was thinking the same thing